flang/lib/Lower/OpenMP.cpp

```diff
@@ -3221,6 +3248,33 @@ static void convertLoopBounds(Fortran::lower::AbstractConverter &converter,
   }
 }
 
+static llvm::SmallVector<const Fortran::semantics::Symbol *> genCodeForIterVar(
```

Comment:
Nit: Maybe it's best to follow the "loop var" name conversion used in other functions like `getLoopVarType` or `createAndSetPrivatizedLoopVar`. Something like this or similar:

```suggestion
static llvm::SmallVector<const Fortran::semantics::Symbol *> genLoopVars(
```